src/App/middlewares/validator.js

```diff
@@ -0,0 +1,25 @@
+import schema  from '../../Database/services/validator';
```

Comment:
⚠️ THIS IS NOT A SERVICE!!!!!!!
Please make sure you move it for your next PR.

Reply:
Actually  already have a validator middleware this is a schema service